packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -41,6 +41,7 @@ class InkResponse extends StatefulWidget {
     this.containedInkWell: false,
     this.highlightShape: BoxShape.circle,
     this.radius,
+    this.borderRadius,
```

Comment:
give this a default value here, then assert in the constructor body that it's not null